pkgs/servers/web-apps/wordpress/packages/plugins.json

```diff
@@ -6,10 +6,10 @@
     "version": "2.4.6"
   },
   "akismet": {
-    "path": "akismet/tags/5.3",
-    "rev": "2991242",
-    "sha256": "1i7y09x2aikndf80wl7xv65m71nv5s0w6jcadpp0dsyzp2m3gjnn",
-    "version": "5.3"
+    "path": "akismet/tags/5.3.1",
+    "rev": "3023195",
+    "sha256": "0ma1d7s535bcjs8x7shzx0qxhxm4cw028rf55l26pkxpw5s7mfvv",
+    "version": "5.3.1"
```

Comment:
What's up with all of these lockfile changes? Is this an update? If so, that should happen in a separate commit and perhaps even a separate PR.